core/src/impl/Kokkos_FunctorAnalysis.hpp

```diff
@@ -401,6 +401,10 @@ struct FunctorAnalysis {
   template <class F, bool is_array = candidate_is_array>
   struct has_join_no_tag_function;
 
+#define KOKKOS_IMPL_LOVATILE  // detect join member functions with
```

Comment:
Typo in the name defined here?

Reply:
No I figured I can call it however I want

Reply:
I lovat it.